books/centaur/sv/svtv/process.lisp

```diff
@@ -1559,30 +1559,60 @@ stvs-and-testing) of the @(see sv-tutorial) for more examples.</p>"
            :hints(("Goal" :in-theory (enable svex-alist-p)))))
 
   (defthm lookup-in-svtv-run-under-iff
-    (iff (assoc key (svtv-run svtv inalist :skip skip :boolvars boolvars :quiet quiet))
+    (iff (assoc key (svtv-run
+                     svtv inalist
+                     :include include :skip skip :boolvars boolvars
+                     :simplify simplify :quiet quiet :readable readable))
          (and (member key (svtv->outs svtv))
-              (not (member key skip))))
+              (if include
+                  (member key include)
+                (not (member key skip)))))
     :hints(("Goal" :in-theory (enable svtv->outs))))
 
   (defthm lookup-in-svtv-run-consp
-    (equal (consp (assoc key (svtv-run svtv inalist :skip skip :boolvars boolvars :quiet quiet)))
-         (and (member key (svtv->outs svtv))
-              (not (member key skip))))
+    (equal (consp
+            (assoc key (svtv-run
+                        svtv inalist
+                        :include include :skip skip :boolvars boolvars
+                        :simplify simplify :quiet quiet :readable readable)))
+           (and (member key (svtv->outs svtv))
+                (if include
+                    (member key include)
+                  (not (member key skip)))
```

Comment:
Maybe rather than an `if`, this should be two theorems with `syntaxp` to help ACL2 choose which one to use. I imagine that one is never going to be reasoning about a call of `svtv-run` which doesn't have at least one of `:include nil` or `:skip nil` in its arguments, and I mean literally `nil` rather than just something provably equal to `nil`.